.github/workflows/build.yml

```diff
@@ -47,6 +47,35 @@ jobs:
             # https://github.com/python/core-workflow/issues/373
             git diff --name-only origin/$GITHUB_BASE_REF.. | grep -qvE '(\.rst$|^Doc|^Misc)' && echo '::set-output name=run_tests::true' || true
           fi
+
+  check_generated_files:
+    name: 'Check if generated files are up to date'
+    runs-on: ubuntu-latest
+    needs: check_source
+    if: needs.check_source.outputs.run_tests == 'true'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Register gcc problem matcher
+        run: echo "::add-matcher::.github/problem-matchers/gcc.json"
```

Comment:
I'm not sure that it's a good idea to get GCC warnings here. Other jobs already produce enough warnings (and the same warnings), no?

Reply:
Yeah, I copied this over from the build job. I was not entirely sure what this step did, but as I was having some issue I just copied everything.